gnd/src/main/java/com/google/android/gnd/persistence/local/room/entity/FeatureEntity.java

```diff
@@ -164,9 +164,9 @@ public static Feature toFeature(FeatureEntity featureEntity, Project project) {
   }
 
   @Nullable
-  public static String listToString(ImmutableList<Point> vertices) {
+  public static String formatVertices(ImmutableList<Point> vertices) {
     if (vertices.isEmpty()) {
-      return null;
+      return "";
```

Comment:
Let's use null in the local db for consistency with Point

Reply:
so return null

Reply:
Yes. We're using null in the local db to represent no value, but empty list in the model objects.